.github/workflows/cmake.yml

```diff
@@ -688,6 +682,18 @@ jobs:
           Get-Command ccache
           (Get-Command ccache).Source
           ccache --show-stats --verbose --version
+      - name: Download Apache Arrow latest source archive and verify via PGP
+        run: |
+          $LATEST_RELEASE_INFO = Invoke-RestMethod "https://api.github.com/repos/apache/arrow/releases/latest"
+          # e.g. "apache-arrow-19.0.0"
+          $APACHE_ARROW_VERSION = $LATEST_RELEASE_INFO.tag_name
+          # "apache-arrow-19.0.0" -> "19.0.0"
+          $ARROW_VERSION = $APACHE_ARROW_VERSION -replace '^apache-', ''
```

Comment:
Can we use better names for them?
`$APACHE_ARROW_VERSION` and `$ARROW_VERSION` are confusing.

Reply:
fix: [1eb3bf6](https://github.com/groonga/groonga/pull/2209/commits/1eb3bf626232bc724b9b371481c8b8a4d29eab0e) The content of `$APACHE_ARROW_VERSION` is actually tag name so I use the name as it is.